data/json/monsters/bird.json

```diff
@@ -111,7 +114,11 @@
     "volume": "5 L",
     "weight": "3750 g",
     "hp": 15,
+    "aggression": 1,
```

Comment:
Perhaps add PLAYER_CLOSE as an anger trigger, but HURT as a fear trigger?

Reply:
Sorry, I didn't have time to answer you, before merge. PLAYER_NEAR_BABY and perhaps HURT flags are good here. You can create new PR, if you want.

Reply:
OK, will take a look at that (need to grade some lab reports before I do anything involved... sigh).